src/System.Net.Primitives/tests/FunctionalTests/CookieCollectionTest.cs

```diff
@@ -108,6 +108,31 @@ public static void Add_CookieCollection_Invalid()
             Assert.Throws<ArgumentNullException>(() => cc.Add((CookieCollection)null));
         }
 
+        [Fact]
+        public static void Clear_Success()
+        {
+            ICollection<Cookie> cc = CreateCookieCollection1();
+            cc.Clear();
+            Assert.Equal(0, cc.Count);
```

Comment:
Nit: to make this a bit more meaningful, I'd suggest adding another assert above the clear that validates the collection isn't already empty prior to calling Clear, e.g.
```C#
Assert.InRange(cc.Count, 1, int.MaxValue);
cc.Clear();
Assert.Equal(0, cc.Count);
```